config/config.go

```diff
@@ -49,3 +50,9 @@ type DBConfig struct {
 	MaxBatchSize      int
 	MaxOpenFiles      int
 }
+
+// GRPCConfig will map the gRPC server configuration
+type GRPCConfig struct {
+	Enable bool
```

Comment:
since it can be `grpc mode` or `stdout` mode exclusively, i think it's better to have a CLI paramter to specify this option

Reply:
done.